core/pom.xml

```diff
@@ -21,7 +21,7 @@
     <fasterxml.jackson.version>2.12.2</fasterxml.jackson.version>
     <jersey.version>2.31</jersey.version>
     <scala.binary.version>2.12</scala.binary.version>
-    <ray.version>1.2.0</ray.version>
+    <ray.version>1.3.0</ray.version>
```

Comment:
Does the package has published? I can't find it: https://mvnrepository.com/artifact/io.ray/ray-api

Reply:
It's published. You can check it now. I saw it [here](https://github.com/ray-project/ray/issues/15496#issuecomment-827298371)